### DISCUSSION TITLE
`git push` and `git pull` using bare repo

### DISCUSSION THREAD

**Discussion by Sogl** [deployphp/deployer#3307] [Help needed]
Sometimes I want to push/pull changes from deployed project, but can't because of these lines in `.git\config`
```
[remote "origin"]
	url = /home/user/public_html/mydomain.com/.dep/repo
	fetch = +refs/heads/*:refs/remotes/origin/*
```

All updates end up in the local repository `.dep/repo`, not to remote `origin`.

How to send to remote?  Maybe there is an option? Tired of constantly changing this `url` line back and forth.

**Comment by antonmedv** [comment]
Maybe dep push may help https://deployer.org/docs/7.x/recipe/deploy/push

**Reply by Sogl** [reply]
And what about pull? I have changes in production (`releases/11`) that I want to send to Bitbucket `origin`. How to do it?

**Reply by antonmedv** [reply]
Don’t know.

**Reply by hansipete** [reply]
Also looking for a way to push changes in release to the remote
